frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/password-login.unit.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import userEvent from "@testing-library/user-event";
+import {
+  createMockSettingDefinition,
+  createMockSettings,
+  createMockTokenFeatures,
+} from "metabase-types/api/mocks";
+import { screen } from "__support__/ui";
+import { setup, SetupOpts } from "./setup";
+
+const setupEnterprise = (opts?: SetupOpts) => {
+  setup({
+    ...opts,
+    tokenFeatures: createMockTokenFeatures({
+      disable_password_login: true,
```

Comment:
It should go to a separate file to be able to test exactly for this token feature